### DISCUSSION TITLE
PTS Taigi

### DISCUSSION THREAD

**Discussion by darrencycheung** [iptv-org/community#367] [Show and tell]
### Channel ID

PTSTaigi.tw

### Stream URL

https://315600.xyz/hatv.php?id=14

### Notes

Found on streamtest.in

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://315600.xyz/hatv.php?id=14 (offline)
<img width="255" alt="image" src="https://user-images.githubusercontent.com/7253922/232247900-e1530196-6749-49be-bb69-36ce9b72c822.png">
